drivers/segger/Kconfig

```diff
@@ -62,6 +62,21 @@ config SEGGER_RTT_BUFFER_SIZE_DOWN
 	---help---
 		Size of the buffer for terminal input to target from host (Usually keyboard input)
 
+choice
+	prompt "SEGGER_RTT_MODE"
+	default SEGGER_RTT_MODE_NO_BLOCK_SKIP
```

Comment:
should we default to SEGGER_RTT_MODE_BLOCK_IF_FIFO_FULL

Reply:
The default option in SEGGER_RTT.c is set to SEGGER_RTT_MODE_NO_BLOCK_SKIP. 

If default to block mode, the board will stuck when debugger disconnected, so we have to build the code again with non-block mode when test on field.